src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -14875,6 +14875,129 @@ public class C
             var comp = CreateCompilation(src, options: TestOptions.DebugExe.WithGeneralDiagnosticOption(ReportDiagnostic.Suppress));
             comp.VerifyDiagnostics();
         }
+
+        [Fact]
+        public void ExperimentalWithWhitespaceDiagnosticID()
+        {
+            var dir = Temp.CreateDirectory();
+            var src = dir.CreateFile("test.cs").WriteAllText("""
+C.M();
+
+[System.Diagnostics.CodeAnalysis.Experimental(" ")]
+public class C
+{
+    public static void M() { }
+}
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    public sealed class ExperimentalAttribute : Attribute
+    {
+        public ExperimentalAttribute(string diagnosticId) { }
+    }
+}
+""");
+            var analyzerConfig = dir.CreateFile(".editorconfig").WriteAllText("""
+[*.cs]
+dotnet_diagnostic.CS9208.severity = warning
```

Comment:
Does this have any effect on the scenario?

Reply:
Or did you mean to attempt to suppress CS9211 instead?

Reply:
Yes, should have been CS9211. Thanks